ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js

```diff
@@ -133,9 +136,50 @@ const defaultHomeBoundingBox = {
   north: 52,
 }
 
+const PanZoomViewBuilder = function(map) {
+  return Marionette.ItemView.extend({
+    template() {
+      return (
+        <React.Fragment>
```

Comment:
`React.Fragment` is only necessary if you have multiple children nodes being returned for your root.